src/components/forms/monthPicker/MonthPicker.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+import Picker from "react-month-picker";
+import InlineError from '../inlineError';
+import moment from 'moment';
+import Label from '../label';
+
+class MonthPicker extends Component {
+  handleAMonthChange = (year, month) => {
+    const {
+      input: {onChange}
+    } = this.props;
+    const momentValue = moment().set({year, month: month - 1, day: 1});
```

Comment:
@mashony Value returned by component doesnt seem to work as expected.
 Expected behaviour: return first date (not day) of month. 
I suspect these params for moment().set function dont work. Also, current version of moment in package.json needs to be updated to support this function